docs/developer_workflow.md

```diff
@@ -1,43 +1,141 @@
 # Developer workflow
 
-## Download source code
-<pre>
-git clone https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix.git
-cd cluster-api-provider-nutanix
-</pre>
-
-## Build source code
-<pre>
-make docker-build
-</pre>
-
-## Create test management cluster
-<pre>
-make kind-create
-</pre>
+This document outlines how to
+
+## Build the source code
+
+1. Download source code:
+
+    ```shell
+    git clone https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix.git
+    cd cluster-api-provider-nutanix
+    ```
+
+1. Build a container image with the source code:
+
+    ```shell
+    make docker-build
+    ```
+
+## Create a local management cluster
+
+1. Create a [KIND](https://kind.sigs.k8s.io/) cluster:
+
+   ```shell
+    make kind-create
+    ```
+
+This will configure [kubectl](https://kubernetes.io/docs/reference/kubectl/) for the local cluster.
+
+## Build a CAPI image
+
+1. Follow the instructions [here](https://image-builder.sigs.k8s.io/capi/providers/nutanix.html#building-capi-images-for-nutanix-cloud-platform-ncp) to build a CAPI image.
```

Comment:
I ~think~ this step is required and its not possible to use an existing image from the PC UI, but let me know if thats not true.

Reply:
This is a required one-off step unless you already happen to have pre-built images that some one else built. we don't distribute the images on PC image store out of the box.

Reply:
to build a CAPI Nutanix ready image